### PR TITLE
Add blog to docs

### PR DESCRIPTION
**Pull request by phil-scott-78**
I want to checkout the output generated from the CI and make sure things don't get squirelly with github actions and playwright. But if that works this might just be ready. 

I'm still not 100% pleased with the index page for the categories, but they still look ok there and aren't going to be very frequently traveled pages anyways.

### CODE REVIEW

**Comment by phil-scott-78** [comment]
Cleaned up some CI errors that popped up then verified the artifact that the docs look good. 

Might be some spelling mistake and the first post isn't exactly pulitzer award winning material, but I feel good about the tech side of things at least :-)

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -21,7 +21,8 @@ jobs:
     - name: Setup dotnet
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 5.0.301
+        dotnet-version: 6.0.x
```
Comment (patriksvensson) [inline .github/workflows/ci.yaml:24]: I think this change will need to be done in `publish.yaml` as well

**Review by patriksvensson — CHANGES_REQUESTED** [review]
Looks good to me, but found one thing that might need to be fixed

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (phil-scott-78) [reply .github/workflows/ci.yaml:24]: oh great catch. got that updated to 6.0.x with the latest push

**Review by patriksvensson — APPROVED** [review]
LGTM 👍

**Comment by patriksvensson** [comment]
Merged! 🎉

**Pull request merged**